config/initializers/business_time.rb

```diff
@@ -0,0 +1,5 @@
+Holidays.between(Date.civil(2019, 1, 1), 2.years.from_now, :gb_eng, :observed).map do |holiday|
+  BusinessTime::Config.holidays << holiday[:date]
+end
+
+BusinessTime::Config.end_of_workday = '11:59 pm'
```

Comment:
I think we might need to start the working day at `0:00` as well, otherwise there's some weirdness about applications submitted before 8am (or whatever the default start of the business day is).